queries/combine_tables.sql

```diff
@@ -0,0 +1,35 @@
+-- Copyright 2020 Google LLC
```

Comment:
Maybe a different name for "combine_tables"? That's a very generic name? Maybe even "combine_clean_tables"? As said in the config comment, maybe some of these queries can be merged?

Reply:
I can change this to combine_clean_tables and edit the .py file as well.

Reply:
Skipped.